crates/bench/src/spacetime_module.rs

```diff
@@ -77,9 +78,8 @@ impl BenchDatabase for SpacetimeModule {
         let module = runtime.block_on(async {
             // We keep a saved database at "crates/bench/.spacetime".
             // This is mainly used for caching wasmtime native artifacts.
-            BENCHMARKS_MODULE
-                .load_module(config, Some(Path::new(env!("CARGO_MANIFEST_DIR"))))
-                .await
+            let path = RootDir(Path::new(env!("CARGO_MANIFEST_DIR")).join(".spacetime"));
+            BENCHMARKS_MODULE.load_module(config, Some(&path)).await
```

Comment:
Ideally, no code outside the handler of 'paths' should construct the layout.

ie: `Paths::from_cargo_manifest`

Reply:
I feel like this case doesn't matter much - to me, this is more or less equivalent to a user passing `--root-dir=$CARGO_MANIFEST_DIR/.spacetime` on the command line. I can add a comment addressing this.